pages/rmrk/detail/_id.vue

```diff
@@ -5,14 +5,45 @@
 <script lang="ts" >
 import { Component, Vue } from 'nuxt-property-decorator'
 import GalleryItem from '@/components/rmrk/Gallery/GalleryItem.vue'
+import createSiteMeta from '@/utils/createSiteMeta'
+import formatBalance from '@/utils/formatBalance'
 
-@Component({
+@Component<GalleryItemPage>({
   name: 'GalleryItemPage',
   components: {
     GalleryItem
+  },
+  head() {
+    const title = this.currentlyViewedItem.title
+    const metaData = {
+      title: title,
+      description: this.currentlyViewedItem.description,
+      url: `https://nft.kodadot.xyz/${this.$route.path}`,
```

Comment:
also old url?

Reply:
reworked completely, url now via baseurl in plugin